WooCommerce/Classes/ViewRelated/Products/Edit Product/Inventory Settings/ProductInventorySettingsViewModel.swift

```diff
@@ -249,7 +249,7 @@ private extension ProductInventorySettingsViewModel {
 
     func createSKUSection() -> Section {
         if let error = error {
-            return Section(errorTitle: error.alertMessage, rows: [.sku])
+            return Section(errorTitle: error.localizedDescription, rows: [.sku])
```

Comment:
nit: Should use `errorDescription` to get the localized message?

Reply:
yes great catch! updated in https://github.com/woocommerce/woocommerce-ios/pull/2781/commits/7c49f0340f93e26c0ede8d41309f61d47213f0a5